docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: "Set Up"
+sidebar_order: 10
+description: "Get started with Sentry's Dev Toolbar, bringing critical Sentry insights and tools into your web app to help your team troubleshoot more effectively."
+---
+
+<Alert level="info">
+  The Dev Toolbar is currently in **beta**. Beta features are still in progress and may have bugs. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-toolbar/issues) if you have any feedback or concerns.
+</Alert>
+
+## Set Up Your Web App
+
+<Alert level="warning">
+    [Enabling tracing](/platforms/javascript/tracing/) is a prerequisite using the Developer Toolbar. Tracing is used to collect page-specific issues and feedback through transactions.
+</Alert>
+
+You need to complete two steps to get the toolbar rendered on the page:
+1. Add or dynamically inject the following script into your web app:
+    ```html
+    <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+    ```
+    It is recommended that you add the script tag at the bottom of the page so it doesn’t block other critical JavaScript.
+
+2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar.
+    ```html
+    <html>
+    <head>...</head>
+    <body>
+        …
+        <script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+        <script>
+        window.SentryToolbar.init({ … });
+        </script>
+    </body>
+    </html>
+    ```
+
+### Unmounting The Toolbar
+
+If you have called `SentryToolbar.init({...})` to render the toolbar, but now want to manually remove or unmount it from the page, you can call the cleanup function that is returned from `init()`. This will unmount all the injected HTML and CSS. Login credentials will not be removed, so you can re-insert the toolbar and still be authenticated.
+```html
+<script src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+<script>
+  const unmountToolbar = window.SentryToolbar.init({ … });
+
+  // sometime later...
+  unmountToolbar();
+</script>
+```
+
+## Init Configuration Fields
```

Comment:
In other parts of the docs I've more commonly seen these referred to as 'options', should we stick to that convention here? I'm also open to 'fields' being the correct term, if that's the case we should update other docs to maintain consistency. 

@lizokm Do you have any thoughts on 'fields' vs 'options'?

Reply:
I've changed it to 'options' to stick with the convention